Stripe/STPEphemeralKeyManager.h

```diff
@@ -0,0 +1,46 @@
+//
+//  STPEphemeralKeyManager.h
+//  Stripe
+//
+//  Created by Ben Guo on 5/9/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "STPEphemeralKeyProvider.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^STPEphemeralKeyCompletionBlock)(STPEphemeralKey * __nullable ephemeralKey, NSError * __nullable error);
+
+@interface STPEphemeralKeyManager : NSObject
+
+/**
+ If the current ephemeral key expires in less than this time interval, a call
```

Comment:
Yeah, we should standardize and make sure all our comments are wrapped – let's punt for a future PR though